src/Pages/HomePage/PageSections/ArrivalsSection.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import '../HomePage.css'
+import NewArrivalCard from '../../../Components/NewArrivalCard/NewArrivalCard'
```

Comment:
As Tanay suggested, it would be better if you export all your components from index.js and then import from only 1 file (index.js)

Reply:
okay will do